frontend/src/app/testResults/submissions/Submission.tsx

```diff
@@ -0,0 +1,87 @@
+import moment from "moment";
+import { useSelector } from "react-redux";
+import { useParams } from "react-router-dom";
+
+import { LoadingCard } from "../../commonComponents/LoadingCard/LoadingCard";
+import { useGetUploadSubmissionQuery } from "../../../generated/graphql";
+
+const Submission = () => {
+  const urlParams = useParams();
+  const internalId = urlParams.id || "";
+
+  const organization = useSelector(
+    (state) => (state as any).organization as Organization
+  );
+
+  const { data: submission, loading, error } = useGetUploadSubmissionQuery({
+    fetchPolicy: "no-cache",
+    variables: {
+      id: internalId,
+    },
+  });
+
+  if (loading) {
+    return <LoadingCard />;
+  }
+
+  if (error) {
+    throw error;
+  }
+
+  const transmissionTimestamp = moment(submission?.uploadSubmission.createdAt);
+  const transmissionDate = transmissionTimestamp.format("DD MMM YYYY");
+  const transmissionTime = transmissionTimestamp.format("k:mm");
+
+  return (
+    <main className="prime-home">
+      <div className="grid-container">
+        <div className="grid-row">
+          <div className="prime-container card-container">
+            {/* Sub-heading */}
+            <div className="usa-card__body text-normal font-body-xs text-base margin-bottom-0">
+              <span>
+                <a href="/results/upload/submissions">
+                  {organization.name} COVID-19 Submissions
+                </a>
+              </span>
+            </div>
+
+            {/* Submission result header */}
+            <div className="usa-card__body">
+              <h1>{[transmissionDate, transmissionTime].join(" ")}</h1>
+            </div>
+
+            {/* Submission detail */}
```

Comment:
ReportStream's submission detail view actually doesn't include the status of the submission - maybe because it only returns successful uploads in the first place? If we would like to include the status here we can do that.

Reply:
that ... is interesting.  Might be worth having a conversation with RS folks about how this is supposed to work.  Because if I'm reading your comment right it seems like there is no way for a submission to be anything other than `PENDING` or `NOT_DELIVERED`.